### PR TITLE
Adding Brazilian programs from bughunt.com.br

### PR DESCRIPTION
**Pull request by toguko**
Adding Brazilian programs from the Bug Bounty platform bughunt.com.br

### CODE REVIEW

File: chaos-bugbounty-list.json [hunk 1]
```diff
@@ -7467,6 +7467,65 @@
 	      "lunarly.com",
 	      "mother-earthproducts.co"
       ]
+    },
+    {
+      "name": "Tim Brasil",
+      "url": "https://admin.bughunt.com.br/program/detail?6058de165576f62d921fc844",
+      "bounty": true,
+      "domains": [
+        "site.tim.com.br"
```
Comment (martinvw) [inline chaos-bugbounty-list.json:7476]: Please leave out all the subdomains

File: chaos-bugbounty-list.json [hunk 2]
```diff
@@ -7467,6 +7467,65 @@
...
+        "www.bitcoinback.com.br"
+      ]
+    },
+    {
+      "name": "OLX Brasil Bug Bounty",
+      "url": "https://admin.bughunt.com.br/program/detail?60ca003869707962894a5b33",
+      "bounty": true,
+      "domains": [
+        "site.tim.com.br",
+        "https://play.google.com/store/apps/details?id=com.schibsted.bomnegocio.androidApp&hl=pt_BR&showAllReviews=true",
```
Comment (martinvw) [inline chaos-bugbounty-list.json:7506]: These should not be included, see also: https://github.com/projectdiscovery/public-bugbounty-programs#-notes

File: chaos-bugbounty-list.json [hunk 3]
```diff
@@ -7467,6 +7467,65 @@
...
+        "link.bitcointrade.com.br",
+        "www.bitcoinback.com.br"
+      ]
+    },
+    {
+      "name": "OLX Brasil Bug Bounty",
+      "url": "https://admin.bughunt.com.br/program/detail?60ca003869707962894a5b33",
+      "bounty": true,
+      "domains": [
+        "site.tim.com.br",
```
Comment (martinvw) [inline chaos-bugbounty-list.json:7505]: This one also appears on line 7476, which seems odd

File: chaos-bugbounty-list.json [hunk 4]
```diff
@@ -7467,6 +7467,65 @@
...
+        "https://play.google.com/store/apps/details?id=com.schibsted.bomnegocio.androidApp&hl=pt_BR&showAllReviews=true",
+        "apigw.olx.com.br/(WIP)",
```
Comment (martinvw) [inline chaos-bugbounty-list.json:7507]: Please leave out paths, just domains

**Review by martinvw — CHANGES_REQUESTED** [review]

**Review by ehsandeep — APPROVED** [review]

**Pull request merged**